### DISCUSSION TITLE
homebrew-cask/Casks/bria.rb can I get older versions?

### DISCUSSION THREAD

**Discussion by mariano-daniel** [Homebrew/discussions#3911] [Getting started]
Is it possible to install older versions of a software? I've read through the man page for brew but couldn't find anything about this.

**Comment by p-linnane** [comment]
Homebrew is a rolling release package manager. We generally only support latest versions. There are some exceptions such as [python@3.10](https://github.com/Homebrew/homebrew-core/blob/master/Formula/python%403.10.rb) vs. [python@3.11](https://github.com/Homebrew/homebrew-core/blob/master/Formula/python%403.11.rb) etc.

**Comment by carlocab** [comment]
You can create your own tap with `brew tap-new` and create your own copy of the cask that installs the version that you need. See `man brew` for details.
